### PR TITLE
Add interactive helper for building Airbyte local CLI commands

### PR DESCRIPTION
**Pull request by ypc-faros**
## Description

Add interactive helper for building Airbyte local CLI commands

**Example for source**
Screenshot 2023-02-23 at 5 25 14 PM

**Example for destination**
Screenshot 2023-02-23 at 5 26 32 PM

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: faros-airbyte-cdk/src/destinations/destination-runner.ts [hunk 1]
```diff
@@ -131,8 +131,8 @@ export class AirbyteDestinationRunner<
 
   specWizardCommand(): Command {
     return new Command()
-      .command('spec-wizard')
-      .description('spec wizard command')
+      .command('airbyte-local-cli-wizard')
+      .description('airbyte local cli wizard command')
```
Comment (tovbinm) [inline faros-airbyte-cdk/src/destinations/destination-runner.ts:135]: `.description('Run a wizard command to prepare argument for Airbyte Local CLI')`

Comment (ypc-faros) [reply faros-airbyte-cdk/src/destinations/destination-runner.ts]: Done, thanks!

File: faros-airbyte-cdk/src/destinations/destination-runner.ts [hunk 2]
```diff
@@ -128,6 +129,31 @@ export class AirbyteDestinationRunner<
...
+          spec.spec.connectionSpecification,
+          [
+            // Prefix argument names with --dst
+            '--dst',
+          ],
+          // Assign section = 0 to the root object's row
+          0
+        );
+        console.log(
+          '\n\nThe arguments below can be used when running this destination' +
```
Comment (tovbinm) [inline faros-airbyte-cdk/src/destinations/destination-runner.ts:150]: `Use the arguments below when running this destination with Airbyte Local CLI (https://github.com/faros-ai/airbyte-local-cli):`

Comment (tovbinm) [reply faros-airbyte-cdk/src/destinations/destination-runner.ts:150]: (same for source)

File: faros-airbyte-cdk/src/destinations/destination-runner.ts [hunk 3]
```diff
@@ -128,6 +129,31 @@ export class AirbyteDestinationRunner<
       });
   }
 
+  specWizardCommand(): Command {
```
Comment (tovbinm) [inline faros-airbyte-cdk/src/destinations/destination-runner.ts:132]: let's rename `specWizardCommand` to `airbyteLocalCLIWizardCommand` here and elsewhere

**Review by tovbinm — APPROVED** [review]

**Comment by tovbinm** [comment]
Awesome stuff!!!

**Comment by ypc-faros** [comment]
> Awesome stuff!!!

Thank you for the feedback 🙇 🥳

**Pull request merged**